### ISSUE TITLE
Debug console is truncating output

### ISSUE THREAD

**Issue by shanebest** [DonJayamanne/pythonVSCode#149]
If I execute/debug a python program, the output in the Visual Studio Code "DEBUG CONSOLE" is limited to 12000 or so characters, then it just stops outputting.

The bug is new with Visual Studio Code version 1.2.0
Bug has been reported to Visual Studio Code, they responded that the bug is with the python module.

Using Windows 10, python 2.7.?

This is great software, keep up the excellent work. Thank you.

**Comment by DonJayamanne** [comment]
Hi, thanks, will look into this some time next week. Sorry for the delay.

**Comment by DonJayamanne** [comment]
Hi, I've managed to replicate the issue.
The first print statement works, after that the debugger just stops working completely!

```python
value = ""
for num in range(0, 15000):
    value = value + str(num)

print value
print "aaa"
```

**Comment by DonJayamanne** [comment]
Similar to #52

**Comment by shanebest** [comment]
Looks like this has magically fixed itself with the release of VS Code 1.2.1
